build.sbt

```diff
@@ -102,15 +102,20 @@ lazy val semanticdbScalacCore = project.in(file("semanticdb/scalac/library")).se
   sharedSettings,
   publishJVMSettings,
   fullCrossVersionSettings,
-  protobufSettings,
   mimaPreviousArtifacts := Set.empty,
   moduleName := "semanticdb-scalac-core",
   description := "Library to generate SemanticDB from Scalac 2.x internal data structures",
-  Compile / unmanagedSourceDirectories ++= {
-    val base = (ThisBuild / baseDirectory).value / "semanticdb"
-    List(base / "metap", base / "cli", base / "semanticdb", base / "metacp", base / "symtab")
-  },
   libraryDependencies += "org.scala-lang" % "scala-compiler" % scalaVersion.value
+).dependsOn(semanticdbShared)
+
+lazy val semanticdbShared = project.in(file("semanticdb/semanticdb")).settings(
+  moduleName := "semanticdb-shared",
+  sharedSettings,
+  publishJVMSettings,
+  fullCrossVersionSettings,
```

Comment:
is this needed? can we publish only once per scala binary version for this project, as well as the new ones?

Reply:
i don't know, i was simply copying from scalac-core module. what will happen if we remove `fullCrossVersionSettings`? cc @tgodzik

Reply:
not sure, I would keep it especially with the recent change to Scala 2.13 standard library. Pretty sure it will break currently if you remove it.

Reply:
https://github.com/scalameta/scalameta/pull/3887